engine/src/xetex_ini.rs

```diff
@@ -15670,7 +15655,8 @@ pub unsafe extern "C" fn tt_run_engine(
     free(font_dsize as *mut libc::c_void);
     free(font_params as *mut libc::c_void);
     free(font_name as *mut libc::c_void);
-    free(font_area as *mut libc::c_void);
+    // free(font_area as *mut libc::c_void);
+    // TODO: how to free a static array, set to null?
```

Comment:
font_area is now a static array here, anyone know how to free it? @cormacrelf @crlf0710

Reply:
You can't free a static array. You need to use Vec or similar if you want the memory on the heap (where it can be freed)

Reply:
I think `font_area` is in stack memory because it's a static slice, so no need to free it?

Reply:
In C a static array would be font_area[size], but font_area is just a static pointer to some memory

Reply:
It's ok to refactor a `*mut str_number` to `[i32; 9001]` as the c2rust demo says:

![image](https://user-images.githubusercontent.com/4054836/66317688-93533180-e94c-11e9-9d79-3ec7f507f565.png)